### DISCUSSION TITLE
helm repo add apollo http://ctripcorp.github.io/apollo/charts 失败404【急】

### DISCUSSION THREAD

**Discussion by shaly** [apolloconfig/apollo#3953] [Q&A] [answered]
- [ ] 我已经检查过[discussions](https://github.com/ctripcorp/apollo/discussions)
- [ ] 我已经搜索过[issues](https://github.com/ctripcorp/apollo/issues)
- [ ] 我已经仔细检查过[FAQ](https://www.apolloconfig.com/#/zh/faq/common-issues-in-deployment-and-development-phase)

**描述bug**
404
PS C:\Users\admin> helm repo add apollo http://ctripcorp.github.io/apollo/charts
helm : Error: looks like "http://ctripcorp.github.io/apollo/charts" is not a valid chart repository or cannot be rea
ched: Get "http://ctripcorp.github.io/apollo/charts/index.yaml": read tcp 172.20.10.2:63549->185.199.111.153:80: wsa
recv: An existing connection was forcibly closed by the remote host.
所在位置 行:1 字符: 1
+ helm repo add apollo http://ctripcorp.github.io/apollo/charts
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (Error: looks li...he remote host.:String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 

**复现**

通过如下步骤可以复现：

1. helm repo add apollo http://ctripcorp.github.io/apollo/charts

**期望**

希望可以add成功

**截图**
![image](https://user-images.githubusercontent.com/20148706/132211863-0b940738-3cbd-4dd9-b18d-236070b6b6a8.png)

如果可以，附上截图来描述你的问题

### 额外的细节和日志

- 版本：
- 错误日志
- 配置：
- 平台和操作系统 linux

**Comment by kezhenxu94** [comment, marked as answer]
See this https://github.com/apolloconfig/apollo/blob/aa9b7ebd01f860c20df9c14129dbbc35cb9d989c/scripts/helm/README.md#3-add-apollo-helm-chart-repository

We moved the repo and updated the domain name
